src/Security.php

```diff
@@ -13,12 +13,12 @@ class Security extends Element
      * Element attribute mustunderstand name
      * @var string
      */
-    const ATTRIBUTE_MUST_UNDERSTAND = 'SOAP-ENV:mustunderstand';
+    const ATTRIBUTE_MUST_UNDERSTAND = ':mustunderstand';
     /**
      * Element attribute mustunderstand name
      * @var string
      */
-    const ATTRIBUTE_ACTOR = 'SOAP-ENV:actor';
+    const ATTRIBUTE_ACTOR = ':actor';
```

Comment:
Let's introduce the constant for outside usage
```suggestion
    const ATTRIBUTE_ACTOR = ':actor';
    /**
     * Envelop namespace
     * @var string
     */
    const ENV_NAMESPACE = 'SOAP-ENV';
```